app/src/main/assets/html/error/secure-connection-failed.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
```

Comment:
these several elements should be wrapped in a `<head>`

Reply:
Cool thanks @cvan for review.  I'll make a PR into @keianhzo with updates.